static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -54,38 +50,27 @@ define([
             // Use Custom Timing to time the googletag code without the sonobi pre-loading.
             performanceLogging.moduleStart(moduleName);
 
-            return new Promise(function(resolve) {
-
-                if (dfpEnv.sonobiEnabled) {
-                    // Just load googletag. Sonobi's wrapper will already be loaded, and googletag is already added to the window by sonobi.
-                    require(['js!googletag.js']);
-                    performanceLogging.addTag('sonobi');
-                } else {
-                    require(['js!googletag.js']);
-                    performanceLogging.addTag('waterfall');
-                }
+            performanceLogging.addTag(dfpEnv.sonobiEnabled ? 'sonobi' : 'waterfall');
 
-                window.googletag.cmd.push = raven.wrap({deep: true}, window.googletag.cmd.push);
+            window.googletag.cmd.push(
+                setListeners,
+                setPageTargeting,
+                moduleEnd
+            );
 
-                window.googletag.cmd.push(
-                    setListeners,
-                    setPageTargeting,
-                    moduleEnd,
-                    resolve
-                );
-            });
+            // Just load googletag. Sonobi's wrapper will already be loaded, and googletag is already added to the window by sonobi.
+            return Promise.resolve(require(['js!googletag.js']));
```

Comment:
Yes 😃  I would even resolve when the request for the script is done, so that the baseline is not impacted by random network conditions. I think it is a good practice to separate module running time from module initialisation.